YesterdayApi/Core/Email/EmailService.cs

```diff
@@ -0,0 +1,30 @@
+using System.Threading.Tasks;
+using MailKit.Net.Smtp;
+using YesterdayApi.Core.Email.Builder;
+using YesterdayApi.Core.Email.Builder.Types;
+
+namespace YesterdayApi.Core.Email
+{
+    public class EmailService : IEmailService
+    {
+        private readonly IEmailDirector _emailDirector;
+
+        public EmailService(IEmailDirector emailDirector)
+        {
+            _emailDirector = emailDirector;
+        }
+
+        public async void SendEmail(IEmailReceiver details, Type type)
```

Comment:
Too complex 🤣. Here i was looking for a builder. Director is another layer which I don't understand the purpose of.

Reply:
You have a answer for that in your previous conversation, scroll up :)